### PR TITLE
User/tsintsir/api login signup

### PR DESCRIPTION
**Pull request by Tsintsir**
I believe this is entirely good to go aside from the lack of authentication.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -74,6 +87,21 @@ def remove_key_from_user(username):
...
+@app.route('/signup/', methods=['POST'])
+def signup():
+    username = request.form.get("username")
+    password = request.form.get("password")
+    email = request.form.get("email")
+    dbw = DatabaseWrapper();
+    try:
+        dbw.create_user(username, password,email)
+    except UsernameAlreadyTakenError as e:
+        return jsonify({"error": "Username is already taken!", "exception": str(e)}), 200
```
Comment (shawkinsl) [inline src/app.py]: Errors shouldn't return 200 codes :)

**Comment by shawkinsl** [comment]
After changing the error 200's to something else, +1

**Comment by LeBat** [comment]
+1

**Comment by sonph** [comment]
lgtm

**Pull request merged**